security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+It is fruitful to analyze how protocol changes like multi-part payments can affect the dynamics of the LN.
+It would be beneficial to explore the temporal nature of the LN graph in more depth.
+
+
+=== Economic incentives and graph structure
+The Lightning graph forms spontaneously.
+Nodes connect to each other based on mutual interest.
+As a result, incentives drive graph development.
+Let's describe some of the relevant incentives:
+
+  * Rational incentives.
+    - Nodes establish channels to send, receive, and route payments (earn fees).
+    - What makes a channel more likely to be established between two nodes that act rationally?
+  * Altruistic incentives.
+    - Nodes establish channels "for the good of the network."
+    - While we should not base our security assumptions on altruism, to a certain extent, altruistic behavior drives Bitcoin (accepting incoming connections, serving blocks).
+    - What role does it play in Lightning?
+
+A recent paper has shown that LN nodes are mostly altruistic.
+Specifically, the earned routing fees do not compensate for the opportunity costs stemming from liquidity lock-up.
+This might change in the future if LN has significantly larger traffic or if a market for routing fees emerge.
+On the other hand, if a node wishes to optimize its routing fees, it would minimize the average shortest path lengths to every other node.
+Put differently, a profit-seeker node will try to locate itself in the _center_ of the channel graph or close to it.
+Given these incentives and the fact that capital is also unevenly distributed, it is unlikely that LN will evolve into a mesh-like network.
+
+= Practical advice for users to protect their privacy =
+
+We're still in the early stages of the Lightning Network.
+Many of the concerns listed in this chapter are likely to be addressed as it matures and grows.
+In the meantime, there are some measures that you can take to guard your node against malicious users; something as simple as updating the default parameters that your node runs with can go a long way in hardening your node.
+
+== Private channels
+If you primarily intend to use your node for personal sends and receives, there is little need to open public channels to the network.
+Since you are not exposing your public channels to the network, you eliminate the risk of a denial-of-service attack on your node.
+
+== Routing considerations
+
+As covered in the <<Denial of service>> section, nodes that open public channels expose themselves to the risk of a series of attacks on their channels.
+While mitigations are being developed on the protocol level, there are many steps that a node can take to protect against denial of service attacks on their public channels.
+
+* Minimum HTLC size: on channel open, your node can set the minimum HTLC size that it will accept.
+Setting a higher value ensures that each of your available channel slots cannot be occupied by a very small payment.
```

Comment:
Another related setting here is `max_htlc`, which can be used to help mitigate probing attacks somewhat, and also let a node control the exposure it'll have to a given payment flow.